layer.md

```diff
@@ -156,3 +156,13 @@ Note that this opaque file will apply to _all_ children, including sub-directori
 Implementations SHOULD generate layers using _explicit whiteout_ files, but MUST accept both.
 
 Any given image is likely to be composed of several of these Image Filesystem Changeset tar archives.
+
+# Foreign Layers
+
+Certain layers, due to legal requirements, may not be regularly distributable.
+Typically, one can download such layers but they should never be uploaded.
```

Comment:
This is an important clarification and frames the stronger language below. I can capitalize the SHOULD or re-phrase it.